src/AppInstallerCLICore/Workflows/FontFlow.cpp

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "FontFlow.h"
+#include "TableOutput.h"
+#include <winget/Fonts.h>
+
+namespace AppInstaller::CLI::Workflow
+{
+    using namespace AppInstaller::CLI::Execution;
+
+    namespace
+    {
+        struct InstalledFontFamiliesTableLine
+        {
+            InstalledFontFamiliesTableLine(Utility::LocIndString familyName, int faceCount)
```

Comment:
Is `LocIndString` sufficient for the family name and face name? If I recall correctly, it is just a wrapper around `std::string`, will this appropriately handle family names that contain Unicode characters? Is it even possible for the names to contain Unicode characters?

Reply:
Yes this should support unicode characters because we are obtaining the localized wstring directly from the fonts library api and converting to UTF8 before creating the LocIndString.